tests/modules/selenium/test_python.py

```diff
@@ -321,6 +321,7 @@ def test_build_script(self):
                             "closeWindow('that_window')",
                             "submitByName(\"toPort\")",
                             "scriptEval(\"alert('This is Sparta');\")",
+                            "rawCode(multi(1)\n  line(2)\n    block(33))",
```

Comment:
let's replace this example with something that won't fail in Python. Like "for+if+print"